wordpress-mobile/WordPress-iOS
# Muriel Colors: Update Image Assets

Refs #11683

This PR updates a bunch of images in our `xcassets` file. It does this by:
- merging `AppImages.xcassets` and `Images.xcassets`
- Creating a duplicate asset collection called `MurielImages.xcassets`
- Switching between the two using the `EXCLUDED_SOURCE_FILE_NAMES` build setting
- Also deletes a few unused assets from the new catalogue because why not

This means that the asset catalogue is chosen irrespective of the `.murielColors` feature flag. Debug builds will use the muriel set, all others the old set.

**These changes can be hard to notice because many of our icons are tinted when used.** Here's some example changes:
<img width="235" alt="Screen Shot 2019-06-25 at 11 38 55 PM" src="https://user-images.githubusercontent.com/517257/60156974-88f89780-97a2-11e9-802f-5291d1f369ee.png">
<img width="109" alt="Screen Shot 2019-06-25 at 11 40 06 PM" src="https://user-images.githubusercontent.com/517257/60156990-96ae1d00-97a2-11e9-820b-e1c60d1b0bea.png">



To test:
- Build as debug, checkout the new icon color, notice a few images around the app have new colors, too.
- Either change the `EXCLUDED_SOURCE_FILE_NAMES` setting or build for release or something, ensure that the old assets are used

Update release notes:
